### DISCUSSION TITLE
参考文献中文括号的问题。

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#175] [Q&A]
你好，
我每天会备份文档。几个月前的文档中，参考文献在文中的括号是`中文`括号。现在我的文档中，参考文献在文中的括号是`英文`括号。我原来以为是我加载宏包造成的，所以我去几个月前的备份文档，其中参考文献在文中的括号是`中文`括号。当我在此运行之前的备份，其参考文献在文中的括号变成了`英文`括号。我时不时会更新Tex Live。 难道是更新Tex Live造成了我们的模板中的参考文献在文中的括号变成了`英文`括号？
否则，同样的文档，为何之前是中文括号，现在运行就是英文括号？
谢谢

**Comment by sikouhjw** [comment]
1. 模板从 v0.6a 开始就使用英文括号；
2. 这个改动是在 https://github.com/whutug/whu-thesis/commit/1157b4cfb638fbc06cbf809e634da3a5a1e30a96 发生的；
3. 在执行重大任务时不要修改生产环境，这是常识；
4. 提问请给 MWE。

**Reply by earthmoon** [reply]
谢谢解答。西文括号与汉字似乎挨得太近了，是加`~`符号，还是有更好的统一解决方式呢？

**Reply by sikouhjw** [reply]
不要管排版，根本不重要。

**Reply by earthmoon** [reply]
也是
